docs/reference/search-application/apis/put-search-application.asciidoc

```diff
@@ -0,0 +1,62 @@
+[role="xpack"]
+[[put-search-application]]
+= Put Search Application API
+[subs="attributes"]
+++++
+
+<titleabbrev>Upsert Search Applications</titleabbrev>
+
+Creates or updates a Search Application.
+
+[[put-search-application-request]]
+== {api-request-title}
+
+`PUT _application/search_application/<name>`
+
+[[put-search-application-prereq]]
+== {api-prereq-title}
+
+Requires the `manage_search_application` cluster privilege.
+Also requires manage `manage` privileges on all indices that are added to the Search Application.
+
+[[put-search-application-path-params]]
+== {api-path-parms-title}
+
+`create`::
+(Optional, Boolean) If true, this request will fail attempting to create a Search Application that already exists.
```

Comment:
This wording is a little unclear to me.

Reply:
I revised, please let me know what you think!

Reply:
👍 💯